roles/openshift_logging/tasks/generate_secrets.yaml

```diff
@@ -56,22 +56,45 @@
   changed_when: no
 
 - name: Generating secrets for elasticsearch
-  command: >
-    {{openshift.common.client_binary}} --config={{ mktemp.stdout }}/admin.kubeconfig secrets new {{secret_name}}
-    key={{generated_certs_dir}}/logging-es.jks truststore={{generated_certs_dir}}/truststore.jks
-    searchguard.key={{generated_certs_dir}}/elasticsearch.jks searchguard.truststore={{generated_certs_dir}}/truststore.jks
-    admin-key={{generated_certs_dir}}/system.admin.key admin-cert={{generated_certs_dir}}/system.admin.crt
-    admin-ca={{generated_certs_dir}}/ca.crt admin.jks={{generated_certs_dir}}/system.admin.jks -o yaml
+  block:
+    - name: Load lib_openshift modules
+      include_role:
+      name: lib_openshift
```

Comment:
lib_openshift should really be a role dependency here, rather than using include_role.